README.md

```diff
@@ -17,7 +17,7 @@ A skeleton app for running your own [EteSync](https://www.etesync.com) server
 Before installing the EteSync server make sure you install `virtualenv` (for **Python 3**):
 
 * Arch Linux: `pacman -S python-virtualenv`
-* Debian/Ubuntu: `apt-get install python3-virtualenv`
+* Debian/Ubuntu: `apt-get install python3-virtualenv python3-venv`
```

Comment:
Maybe it makes sense to split Debian and Ubuntu? It looks like they are just different here (though I find it odd!).

Reply:
yep, good idea